tomcat/src/main/java/org/apache/catalina/handler/RequestMappingHandler.java

```diff
@@ -0,0 +1,58 @@
+package org.apache.catalina.handler;
+
+import java.util.UUID;
+import org.apache.catalina.session.Session;
+import org.apache.catalina.session.SessionManager;
+import org.apache.coyote.http11.ContentType;
+import org.apache.coyote.http11.HttpRequest;
+import org.apache.coyote.http11.HttpResponse;
+import org.apache.coyote.http11.HttpStatusCode;
+
+public class RequestMappingHandler {
+
+    public HttpResponse request(HttpRequest request) {
+        if (request.getPath().equals("/")) {
+            HttpResponse response = new HttpResponse(HttpStatusCode.OK, ContentType.HTML, request.getPath());
+            setSession(request, response);
+            return response;
+        }
```

Comment:
정말 사소하지만.. 아래처럼 두고 맨 밑에서 한번에 setSession과 return을 호출하면 중복된 코드가 많이 줄 것 같아요
```suggestion
        HttpResponse response = new HttpResponse(HttpStatusCode.NOT_FOUND, ContentType.HTML, "/404.html");
        if (request.getPath().equals("/")) {
            response = new HttpResponse(HttpStatusCode.OK, ContentType.HTML, request.getPath());
        }
```

Reply:
안그래도 너무 중복이어서 어떻게 최적화할지 고민이었는데 딱 좋은 방법인 것 같아요!
피드백 감사합니다! 반영하겠습니다 :)